src/crc-start.ts

```diff
@@ -47,31 +49,41 @@ export async function startCrc(
         await needSetup();
       } catch (error) {
         logger.error(error);
+        provider.updateStatus('stopped');
```

Comment:
If `error` display `stopped` ?
hmmmm... did we do this for the tray too?

Reply:
I'm not sure that PD render/behaves correctly when provider reported `error` state, and user has a way to recover from error state